src/containers/RecentTrades/Market.tsx

```diff
@@ -49,91 +30,75 @@ const handleHighlightValue = (prevValue: string, curValue: string) => {
     );
 };
 
+const RecentTradesMarket = () => {
+    const { formatMessage } = useIntl();
+    const dispatch = useDispatch();
 
-class RecentTradesMarketContainer extends React.Component<Props> {
-    public componentDidMount() {
-        if (this.props.currentMarket) {
-            this.props.tradesFetch(this.props.currentMarket);
-        }
-    }
-
-    public componentWillReceiveProps(next: Props) {
-        if (next.currentMarket && this.props.currentMarket !== next.currentMarket) {
-            this.props.tradesFetch(next.currentMarket);
-        }
-    }
-
-    public shouldComponentUpdate(nextProps: Props) {
-        return JSON.stringify(nextProps.recentTrades) !== JSON.stringify(this.props.recentTrades);
```

Comment:
Is it updating in proper way after deleting ```shouldComponentUpdate()```?

Reply:
it is, let's check together